behaviour_and_protocol/consulting_101.md

```diff
@@ -1,26 +1,100 @@
+* **Explain system issues, highlighting the reasons for unexpected outcomes and the steps taken to fix these.**
+  * Example: "The issue occurred because the user could input text, and we were expecting numbers. We've updated the input and added additional tests."
+
+Etiquette and Professionalism
+-----------------------------
+
+* **Punctuality for meetings.**
+  * Some meetings require more preparation than others. Make sure to prepare for the meeting.
+  * Example: Arrive 5-10 minutes early to setup and review meeting materials.
+* **Stay professional in both physical and digital interactions.**
+  * You do not need formal attire for client meetings, but you should be presentable.
+  * Your messages are a part of your professional image. Make sure they are thought-through and polite.
+* **Ensure a professional setup for remote interactions.**
+  * Example: Ensure the background is tidy and well-lit for video meetings.
+* **Keep track of time spent on projects accurately and consistently.**
+  * Log your hours daily to maintain accurate project timelines and budgeting.
+
+Proactiveness
+-----------------------
+
+* **Be proactive in identifying and addressing potential issues or enhancements.**
+  * If you find something that could be fixed, could be optimized, or could be updated in other ways, communicate this to the team.
+  * Example: "I noticed a strange pattern in the data and wanted to make sure that this is not an error or would otherwise impact the project."
+* **Engage in proactive communication while being mindful of the implementation's impact.**
+  * Example: Before implementing a change, discuss potential impacts with the client and team.
+
+Solution-Oriented Approach
+--------------------------
+
+* **Avoid outright denials when addressing client requests; instead, propose alternative solutions or phased approaches.**
+  * Avoid: "That's impossible."
+  * Try: "That's challenging, but let's explore some alternatives that might work."
+  * Avoid: "It will take too much time."
+  * Try: "This is a complex feature. If this is critical for the project, we can research and estimate it before we move on."
+  * Avoid: "Do you really need that?"
+  * Try: "Let us put it in the ice box and assess it when we have finished the business critical features."
+* **Suggest a Proof of Concept (POC) for technically challenging or untested solutions to demonstrate if they're feasible.**
+  * Example: "We can develop a small POC to test this approach within a week."
+
+Understanding the Client’s Perspective
+-------------------------------------------
+
+* **View and assess the platform from the client's perspective to better understand their needs.**
+  * You might find new features or improvements that the client has not thought of.
+  * Example: Regularly use the client's platform yourself to identify areas for improvement.
```

Comment:
As a developer I'm unsure about what is expected of me here.

Should I regularly just randomly click around the project I'm working on to identify problems?

Reply:
Good point. 
I have had projects where the team dedicated time to use the system, but it's not a normal thing currently

Reply:
Do you have any thoughts on how we could clarify?

Reply:
Good point. I've added your idea with "Test Tuesday"